exercises/practice/quadratic-roots/.docs/instructions.md

```diff
@@ -0,0 +1,9 @@
+# Instructions
+
+Use Z3 to implement a function that takes numbers `a, b, and c` that
```

Comment:
I don't think that you should say "Use Z3 to ..." since this will be under the Z3 track. Maybe just start the sentence with "Implement ..."